skbeam/core/roi.py

```diff
@@ -430,18 +476,27 @@ def circular_average(image, calibrated_center, threshold=0, nx=100,
     ring_averages : array
```

Comment:
Ignore counts above `threshold`
This should be:
         Ignore counts below `threshold`